pipeline/errors/error_redirect.go

```diff
@@ -13,8 +14,9 @@ var _ Handler = new(ErrorRedirect)
 
 type (
 	ErrorRedirectConfig struct {
-		To   string `json:"to"`
-		Code int    `json:"code"`
+		To       string `json:"to"`
+		Code     int    `json:"code"`
+		UrlParam string `json:"url_param"`
```

Comment:
In go, names are usually uppercased like this.
```suggestion
		URLParam string `json:"url_param"`
```